OpenRA.Platforms.Default/Sdl2GraphicsDevice.cs

```diff
@@ -75,7 +75,9 @@ public Sdl2GraphicsDevice(Size windowSize, WindowMode windowMode)
 			}
 
 			context = SDL.SDL_GL_CreateContext(window);
-			SDL.SDL_GL_MakeCurrent(window, context);
+			if (context == IntPtr.Zero || SDL.SDL_GL_MakeCurrent(window, context) < 0)
```

Comment:
Does SDL.SDL_GL_MakeCurrent return an unsigned integer? If not, should this be != 0?